test/markup/powershell/misc.txt

```diff
@@ -19,6 +19,7 @@ enum MediaTypes {
 "Good Dog" -match "Dog"
 2 -eq 2
 "abc" -ne "def"
+"abc" --ne "def"
```

Comment:
This doesn't test anything thought because `--ne` is an operator, not a literal... so it was already highlighted.

Reply:
Just make something up.  `--format-drive-fully`, etc...

Reply:
Can you suggest to me how to write the test case?

Reply:
```
format /dev/devices/driveA --format-drive-fully -yes
```

Reply:
<span class="hljs-literal">--match</span><span class="hljs-literal">-lower</span><span class="hljs-literal">-case</span>

Should the result be like this ?

Reply:
Separate span tags right?  @joshgoebel

Reply:
That looks right, but you'll know when the tests pass.